server/php-engine-vars.h

```diff
@@ -21,6 +21,8 @@
 #define DEFAULT_SCRIPT_TIMEOUT 30
 #define MAX_SCRIPT_TIMEOUT (60 * 7)
 
+enum class ProcessType { master, general_worker, job_worker };
```

Comment:
Let's distinguish `rpc_worker` and  `http_worker` instead of single `general_worker`

We can do this by checking that `rpc_clients` are not empty. In that case it's `rpc_worker`, otherwise - `http_worker`
https://github.com/VKCOM/kphp/blob/2c3594f04c432dd262f2cc38efadb7e59fde5501/server/lease-rpc-client.h#L24